cirq-google/cirq_google/ops/coupler_pulse.py

```diff
@@ -0,0 +1,110 @@
+# Copyright 2020 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Any, Optional, Tuple
+
+import numpy as np
+
+import cirq
+
+
+_MIN_DURATION = cirq.Duration(nanos=0)
+_MAX_DURATION = cirq.Duration(nanos=100)
+
+
+@cirq.value_equality(approximate=True)
+class CouplerPulse(cirq.ops.gate_features.TwoQubitGate):
+    """Tunable pulse for entangling adjacent qubits.
+
+    For experimental usage only.
```

Comment:
Can we move this to cirq_google/experimental/ops?

Reply:
Done.